graph.go

```diff
@@ -195,16 +195,21 @@ func (c *GraphCmd) Execute(args []string) error {
 }
 
 func relPath(base, path string) string {
-	rp, err := filepath.Rel(base, path)
+	pathWithoutSymlinks, err := filepath.EvalSymlinks(path)
```

Comment:
I'll check that. If yes, the solution might be to also evaluate syminks in `base` and then calculate the relative path between those two resolved paths. To me, evaluating syminks sounds better than special-casing `.srclib-gopath`.